solr/config/configsets/all_collection_oag56_v1/managed-schema

```diff
@@ -583,6 +587,7 @@
   <field name="standards" type="strings" docValues="true" indexed="true" stored="false" useDocValuesAsStored="true"/>
   <field name="status" type="string" docValues="true" indexed="true" stored="false" useDocValuesAsStored="true"/>
   <field name="status_monitoring_url" type="text_general" sortMissingLast="true" required="false"/>
+  <field name="subject" type="text_general"/>
```

Comment:
Have you added this field by yourself? Is it a necessary field? It seems not. If that is the case, delete it.

Reply:
It was already in the schema which I copied. I removed it.